### DISCUSSION TITLE
Where are validation metrics stored?

### DISCUSSION THREAD

**Discussion by SilverStarCoder** [open-edge-platform/anomalib#1765] [Q&A]
I am following the tutorial notebooks, but have not been able to find where the actual test results are stored. For example, after running:
![image](https://github.com/openvinotoolkit/anomalib/assets/141123187/5dcc30b2-3a9e-44fd-939b-a0e41669b8b9), nothing is returned, and I don't see anything created in the working directory. I believe a results folder should be created containing the images with heatmaps and scores, but nothing happens.. Would appreciate some guidance; is there a setting that the tutorial notebooks are missing ? Seems kind of pointless to do validation if you can't see any metrics or results.

**Comment by samet-akcay** [comment]
@SilverStarCoder, this is related to https://github.com/openvinotoolkit/anomalib/issues/1747, I think. 
I'll try to add a fix this week.

**Reply by SilverStarCoder** [reply]
Thanks for the update @samet-akcay . What about the validation images (original, heatmap, segmentation, etc..)? Will these be automatically saved in the working directory, or is there a specific path that can get inputted somewhere for these to get saved into?

**Reply by samet-akcay** [reply]
Yeah, `ImageVisualizer` will ensure that all these handled properly. I'll try to fix this and update the notebooks.
https://github.com/openvinotoolkit/anomalib/blob/78e0ace50bf147106354e5be3d99c7245fa8aee9/src/anomalib/utils/visualization/image.py#L72
